src/osd/PrimaryLogPG.cc

```diff
@@ -4378,6 +4378,9 @@ void PrimaryLogPG::log_op_stats(const OpRequest& op,
   const utime_t process_latency = now - op.get_dequeued_time();
 
   osd->logger->inc(l_osd_op);
+  if (latency >= g_conf()->osd_op_complaint_time) {
```

Comment:
most of places I've checked use op.get_initiated() as a starting point for complaint time. And you use m->get_recv_stamp() call above to set latency var instead.  For the sake of uniformity may be use get_initiated() here as well?

Reply:
Updated.